resources/js/composables/selections/selections.ts

```diff
@@ -75,15 +75,19 @@ export function useSelection(
 	}
 
 	function handlePhotoCtrl(idx: number, e: Event): void {
+		console.log("CTRL", idx);
 		if (isPhotoSelected(idx)) {
+			console.log("REMOVE");
 			removeFromPhotoSelection(idx);
 		} else {
+			console.log("ADD");
 			addToPhotoSelection(idx);
 		}
 		lastPhotoClicked.value = idx;
 	}
 
 	function handlePhotoShift(idx: number, e: Event): void {
+		console.log("SHIFT", idx);
```

Comment:
Are the console logs meant to be here?

Reply:
Fixed : [82d7b3d](https://github.com/LycheeOrg/Lychee/pull/3252/commits/82d7b3d731213b9ba3aa9211e0d3535a1df3adac)

Reply:
[56cc6a1](https://github.com/LycheeOrg/Lychee/pull/3252/commits/56cc6a1027674d5502a062b3e3507fb75072fd2b)